op-program/Makefile

```diff
@@ -46,16 +46,15 @@ check-custom-chains: op-program-host
 	./bin/op-program configs check-custom-chains
 
 reproducible-prestate: check-custom-chains
-	@docker build --output ./bin/ --progress plain -f Dockerfile.repro.next ../
 	@docker build --output ./bin/ --progress plain -f Dockerfile.repro ../
 	@echo "-------------------- Production Prestates --------------------\n"
 	@echo "\nCannon64 Absolute prestate hash: "
 	@cat ./bin/prestate-proof-mt64.json | jq -r .pre
 	@echo "\n-------------------- Experimental Prestates --------------------\n"
 	@echo "CannonInterop Absolute prestate hash: "
 	@cat ./bin/prestate-proof-interop.json | jq -r .pre
-	@echo "\nCannon64Next Absolute prestate hash: "
-	@cat ./bin/prestate-proof-mt64Next.json | jq -r .pre
+#	@echo "\nCannon64Next Absolute prestate hash: "
+#	@cat ./bin/prestate-proof-mt64Next.json | jq -r .pre
```

Comment:
I have another commit to push up that uncomments this so we continue to produce the "next" prestate.  Will push it after e2e tests finish running.